site/gdocs/SDGTableOfContents.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState } from "react"
+import { TocHeading } from "../../clientUtils/owidTypes.js"
+import { faArrowDown } from "@fortawesome/free-solid-svg-icons/faArrowDown"
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome/index.js"
+import { faPlus } from "@fortawesome/free-solid-svg-icons/faPlus"
+import { faMinus } from "@fortawesome/free-solid-svg-icons/faMinus"
+import classNames from "classnames"
+import AnimateHeight from "react-animate-height"
+
+const VERTICAL_TAB_CHAR = "\u000b"
+
+export default function SDGTableOfContents({ toc }: { toc: TocHeading[] }) {
+    const [height, setHeight] = useState<"auto" | 0>(0)
+    const [isOpen, setIsOpen] = useState(false)
+
+    const toggleIsOpen = () => {
+        setHeight(height === 0 ? "auto" : 0)
+    }
+
+    const tocHeadingsWithSupertitle = toc.map((heading) => {
+        const [beforeSeparator, afterSeparator] =
+            heading.text.split(VERTICAL_TAB_CHAR)
+
+        return {
+            ...heading,
+            supertitle: afterSeparator ? beforeSeparator : undefined,
+            title: afterSeparator || beforeSeparator,
+        }
+    })
+
+    return (
+        <nav className={classNames("sdg-toc", { open: isOpen })}>
```

Comment:
Thanks for bringing that up!

I initially had mixed feelings about nav + role button, but it seems ok (at least I couldn't find anything that said otherwise).

I've updated the code mixing your suggestions with this recommendation: https://w3c.github.io/aria-practices/examples/menu-button/menu-button-links.html

One thing I haven't done is change the role of the nav element. According to https://www.w3.org/TR/wai-aria-1.0/roles:

> Roles are element types and authors MUST NOT change role values over time or with user actions.

Let me know you have a different reading, and if you see ways to improve on this iteration. Thanks!